src/main/java/de/tum/in/www1/artemis/domain/plagiarism/PlagiarismMatch.java

```diff
@@ -66,6 +68,24 @@ public void setLength(int length) {
         this.length = length;
     }
 
+    @Override
```

Comment:
please add a comment why `equals` is needed here. In other domain classes we avoid it, in particular when they inherit from DomainObject, because then, the id of the database is the unique identifier and DomainObject already offers the equals method. 

For some developers, it's apparently unclear when equals is needed and when not.